interactions/api/models/gw.py

```diff
@@ -796,6 +796,9 @@ class MessageReaction(DictSerializerMixin):
     member: Optional[Member] = field(converter=Member, default=None)
     emoji: Optional[Emoji] = field(converter=Emoji, default=None)
 
+    def __attrs_post_init__(self):
+        self.member._client = self._client
```

Comment:
discord doesn't send `member` data in `message_reaction_remove` event so it will raise error `An error occurred dispatching message_reaction_remove: 'NoneType' object has no attribute '_client'`.
Use `add_client=True`